data/ssystem_major.ini

```diff
@@ -1950,13 +1950,13 @@ color=0., 0., 0.
 halo=false
 hidden=true
 name=Earth Observer
-orbit_Inclination=90
-orbit_MeanLongitude=90
-orbit_Period=100000000000
+; orbit_Inclination=90
```

Comment:
The final question about this PR is whether we can already finalize this and remove the define in Orbit.h to go back to the previous Observer solution. 

For this, we should probably think about another solution regarding location. Currently I dislike that the Location panel shows the "nomap.png" planet texture. We could add a decorative "observer" texture. Also, the 90° remains when going to another planet/moon. Is there a way to just deliver 90/0 without changing the other locations' latitude/longitude?

Reply:
> Currently I dislike that the Location panel shows the "nomap.png" planet texture.

I think at least the map should be removed, since geographic location doesn't make sense for observers. A decorative placeholder can indeed be a nice replacement.

> Also, the 90° remains when going to another planet/moon.

I don't reproduce this. When I switch from Earth/Astana to Solar System Observer and back to Earth, I return to the coordinates of Astana. The _Latitude_ and _Longitude_ spinboxes in the _Location_ dialog keep their values all the time, without changing to 90°. The location name (and its properties like light pollution) is lost though.

Reply:
The coordinates of Astana remain? Is this Qt6 on Linux? These are the stupid little details between OSes :-(
I have not tried what happens if I disable and hide the position UI elements as soon as I switch to an "observer" planet. 

I could add planet close-up screenshots for each observer. Or do you have better ideas (and actual images)? Some spaceship cockpit?

Reply:
Yes it's Qt6 on Linux. Building Qt5 version now, but I don't think it would differ.

As for the artwork, I don't have any good ideas on it. Planet close-up seems a bit misleading, as if we go to this planet. Spaceship cockpit might be OK, though it shouldn't look like a high-detail something that would induce too high expectations.

Reply:
Indeed, Qt5 version works the same way.

Reply:
We actually _are_ going to this planet. How much the closeup goes will be up to the user. Adding orbits will help getting the idea. However, Saturn and Jupiter have those overlapping orbits. May need to paint them away. And yes, I will not paint a cockpit.